data/categoryRepository.go

```diff
@@ -286,30 +294,9 @@ func (r *CategoryRepository) GetCategoriesByName(name string) (categories []mode
 		categories = append(categories, models.Category{
 			Id:       category.Id,
 			Name:     category.Name,
+			Universe: category.Universe,
 			ParentId: parentId,
 		})
 	}
 	return
 }
-
-func (r *CategoryRepository) GetChildrenOf(id int64) (categories []models.Category, err error) {
```

Comment:
FYI: This is referenced by GetCategoryPackage() in `categoryController.go`.

Reply:
Actually that's on a different branch (I think you might have got mixed up if you were switching back and forth between them). No sooner did I delete the method here than I needed it again and recreated it on the other branch :)